RELEASE-NOTES.txt

```diff
@@ -11,7 +11,7 @@
 
 17.8
 -----
-
```

Comment:
Should we move the release note to new number since 17.8 is released?

Reply:
Good catch! Updated in https://github.com/woocommerce/woocommerce-ios/pull/12268/commits/6976a950042b05bda0746a0072c392cd5dfc9727